src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+        options.merge(safariOptions);
+
+
+        options.setCapability("browser", "Safari");
+        options.setCapability("browser_version", "latest");
+        options.setCapability("os", "OS X");
+        options.setCapability("os_version", "Big Sur");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Edge-Safari");
+        options.setCapability("name", "WebRTC Room Joining - Safari Browser");
+        options.setCapability("browserstack.idleTimeout", 300);
+
+        return options;
+    }
+    
+    public static ChromeOptions getiOSConfiguration() {
+
+        SafariOptions safariOptions = new SafariOptions();
+        safariOptions.setUseTechnologyPreview(true);
+
+        ChromeOptions options = new ChromeOptions();
+        options.merge(safariOptions);
+
+
+        options.setCapability("device", "iPhone 12");
+        options.setCapability("real_mobile", "true");
+        options.setCapability("os_version", "14");
+        options.setCapability("autoAcceptAlerts", "true");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Android-iOS");
+        options.setCapability("name", "WebRTC Room Joining - iOS");
+        options.setCapability("browserstack.idleTimeout", 300);
+
+        return options;
+    }
+    
+    public static ChromeOptions getAndroidConfiguration() {
+
+    	ChromeOptions options = new ChromeOptions();
+
+        options.addArguments(
+              "--use-fake-device-for-media-stream",
+                "--use-fake-ui-for-media-stream"
+        );
+
+        options.setCapability("os_version", "11.0");
+        options.setCapability("device", "Samsung Galaxy S21");
+        options.setCapability("real_mobile", "true");
+        options.setCapability("autoGrantPermissions", "true");
+        options.setCapability("build", "WebRTC Dummy Video Call Build - Android-iOS");
+        options.setCapability("name", "WebRTC Room Creation - Android");
+        options.setCapability("browserstack.idleTimeout", 300);
+        
+        return options;
+    }
+   
+   public static void main(String[] args) {
+	   	//Disabling Selenium messages.
+	   	java.util.logging.Logger.getLogger("org.openqa.selenium").setLevel(Level.OFF);
+	   	
+        try {
+           //Generating Random new RoomID
+           int roomId = getRoomID();
+            
+        	//Getting User's Selection
+        	Scanner sc = new Scanner(System.in);
+        	System.out.println("Please Select a Browser Combination to initiate the WebRTC test on BrowserStack : ");
+        	System.out.println("1 : Press 1 to initiate WebRTC test on Google Chrome and Firefox Browser Combination ");
+        	System.out.println("2 : Press 2 to initiate WebRTC test on Edge and Safari Browser Combination ");
+        	System.out.println("3 : Press 3 to initiate WebRTC test on Android and iOS Combination");
+        	int userSelection  = sc.nextInt();
+        	sc.close();
+        	Thread t1 = null;
+        	Thread t2 = null;
```

Comment:
please have more appropriate naming for these thread objects

Reply:
done